src/main/java/com/FoodyBuddy/Dao/OrderDAO.java

```diff
@@ -0,0 +1,16 @@
+package com.FoodyBuddy.Dao;
+
+import java.util.List;
+
+import com.FoodyBuddy.Model.Order;
+
+public interface OrderDAO {
+
+	public void save(Order c);
+	public List<Order> getListByBuyerId(Integer buyerId);
+	public Order getByOrderId(Integer orderId);	
+	public Order getById(Integer id);
+	public void update(Order order);	
+	public void delete(Order order);
```

Comment:
While Implementing Buyer, we deleted an Object using id. So we need to come up with a uniform solution.